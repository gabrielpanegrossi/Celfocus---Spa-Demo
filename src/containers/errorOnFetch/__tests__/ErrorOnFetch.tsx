import { render, screen } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import ErrorOnFetch from '../index';

describe('ContainerHeader', () => {
  beforeEach(() => {
    render(
      <Router>
        <ErrorOnFetch />
      </Router>
    );
  });

  it('should match snpashot', () => {
    const container = render(
      <Router>
        <ErrorOnFetch />
      </Router>
    );
    expect(container).toMatchSnapshot();
  });

  it('should render', () => {
    expect(screen.queryByRole('heading')).toBeInTheDocument();
    expect(screen.queryByLabelText('error-message')).toBeInTheDocument();
  });
});
