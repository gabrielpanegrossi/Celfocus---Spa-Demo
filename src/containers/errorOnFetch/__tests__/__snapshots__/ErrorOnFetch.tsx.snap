// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerHeader should match snpashot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-gKXOVf bGBQSC"
      >
        <header
          class="sc-bczRLJ bXXmoA"
        >
          <div
            class="sc-gsnTZi jGtclP"
          >
            <button
              class="sc-dkzDqf dUlZGR"
            >
              Back
            </button>
          </div>
          <div
            class="sc-gsnTZi jGtclP"
          >
            <h1
              class="sc-hKMtZM cfLWee"
            >
              Oops
            </h1>
          </div>
          <div
            class="sc-gsnTZi jGtclP"
          />
        </header>
        <div
          class="sc-iBkjds fzLwdz"
        >
          <p
            aria-label="error-message"
            class="sc-ftvSup bhvCTh"
          >
            Seems that we have a problem communicating to the api server, please make sure you've set the correct url on the env file and try again.
          </p>
        </div>
      </div>
    </div>
    <div>
      <div
        class="sc-gKXOVf bGBQSC"
      >
        <header
          class="sc-bczRLJ bXXmoA"
        >
          <div
            class="sc-gsnTZi jGtclP"
          >
            <button
              class="sc-dkzDqf dUlZGR"
            >
              Back
            </button>
          </div>
          <div
            class="sc-gsnTZi jGtclP"
          >
            <h1
              class="sc-hKMtZM cfLWee"
            >
              Oops
            </h1>
          </div>
          <div
            class="sc-gsnTZi jGtclP"
          />
        </header>
        <div
          class="sc-iBkjds fzLwdz"
        >
          <p
            aria-label="error-message"
            class="sc-ftvSup bhvCTh"
          >
            Seems that we have a problem communicating to the api server, please make sure you've set the correct url on the env file and try again.
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-gKXOVf bGBQSC"
    >
      <header
        class="sc-bczRLJ bXXmoA"
      >
        <div
          class="sc-gsnTZi jGtclP"
        >
          <button
            class="sc-dkzDqf dUlZGR"
          >
            Back
          </button>
        </div>
        <div
          class="sc-gsnTZi jGtclP"
        >
          <h1
            class="sc-hKMtZM cfLWee"
          >
            Oops
          </h1>
        </div>
        <div
          class="sc-gsnTZi jGtclP"
        />
      </header>
      <div
        class="sc-iBkjds fzLwdz"
      >
        <p
          aria-label="error-message"
          class="sc-ftvSup bhvCTh"
        >
          Seems that we have a problem communicating to the api server, please make sure you've set the correct url on the env file and try again.
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
